via/views/api/youtube.py

```diff
@@ -1,1328 +1,42 @@
-from pyramid.httpexceptions import HTTPNotFound
+import logging
+
+from h_pyramid_sentry import report_exception as report_exception_to_sentry
 from pyramid.view import view_config
 
+from via.services import YouTubeService
+
+logger = logging.getLogger(__name__)
+
 
 @view_config(route_name="api.youtube.transcript", permission="api", renderer="json")
 def transcript(request):
```

Comment:
:+1: https://github.com/hypothesis/via/pull/1010/commits/fedb4b9baed97d491a638051c553af25fbd43c07

Yeah I agree with this: the `{noun}_{verb}()` method/function naming pattern is nice and clear, and it also makes space for a single URL like `/api/youtube/transcript/{video_id}` to have different views for different HTTP methods like `get_transcript()`, `put_transript()`, `patch_trancript()` and `delete_transcript()`.

This also reminded me that the current API only supports `GET`, so I added a predicate for that so that it no longer accepts other HTTP verbs (even `DELETE`) and just responds by getting and returning the transcript :) https://github.com/hypothesis/via/pull/1010/commits/a6fe7397607fd72c4aa685b10882b1465c92d282